app/Http/Controllers/ContestsController.php

```diff
@@ -49,10 +50,15 @@ public function create()
      */
     public function store(ContestRequest $request)
     {
-        $dateParams = ['signup_start_date', 'signup_end_date'];
```

Comment:
Well, the form now includes the new messaging inputs as well, so it was no longer `except()ing` just date param. Was pretty cool though, but with the changes, no longer made sense :wink: